### PR TITLE
[Do-Not-Merge]Generating SDK from extended loc swagger specs

### PR DESCRIPTION
**Pull request by pavanrachapudy**
Generating SDK for extended location swagger specs into compute 2020-12 branch.
Added Do Not Merge Tag as tests need to be added for this change. Since it's a change to new api version, other files with api change are also being shown up in the committed files. This PR is just to make sure the SDK can be generated from Swagger Specs.

### CODE REVIEW

**Comment by pavanrachapudy** [comment]
@Sandido @viananth

File: sdk/compute/Microsoft.Azure.Management.Compute/src/Generated/Models/ExtendedLocation.cs [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        /// Gets or sets the name of the extended location.
+        /// </summary>
+        [JsonProperty(PropertyName = "name")]
+        public string Name { get; set; }
+
+        /// <summary>
+        /// The type of the extended location.
+        /// </summary>
+        [JsonProperty(PropertyName = "type")]
+        public static string Type { get; private set; }
```
Comment (DwayneNeed) [inline sdk/compute/Microsoft.Azure.Management.Compute/src/Generated/Models/ExtendedLocation.cs:62]: I don't understand why this is static.  What did we do in the swagger to cause it to generate this way?  Both the type and the name should be instance properties.

**Review by DwayneNeed — CHANGES_REQUESTED** [review]

**Comment by jsquire** [comment]
Hi @pavanrachapudy.  There hasn't been recent engagement on this PR.  Would you please be so kind as to let us know if this is still an active work stream or if the PR should be closed out?

**Comment by Sandido** [comment]
@jsquire , There have been a couple of blockers preventing the underlying Rest API from releasing as scheduled. This PR will be used in the near future. Please leave it as open for now.

**Comment by YalinLi0312** [comment]
@pavanrachapudy If no updates, will close it next week. Thanks

**Pull request closed without merging**